contracts/sources/obligation.move

```diff
@@ -1264,13 +1324,54 @@ module suilend::obligation {
         )
     }
 
+    #[test_only]
+    fun ausd_reserve<P>(scenario: &mut Scenario): Reserve<P> {
+        let config = default_reserve_config();
+        let builder = reserve_config::from(&config, test_scenario::ctx(scenario));
+        reserve_config::set_open_ltv_pct(&mut builder, 50);
+        reserve_config::set_close_ltv_pct(&mut builder, 80);
+        reserve_config::set_max_close_ltv_pct(&mut builder, 80);
+        reserve_config::set_borrow_weight_bps(&mut builder, 20_000);
+        reserve_config::set_interest_rate_utils(&mut builder, {
+            let v = vector::empty();
+            vector::push_back(&mut v, 0);
+            vector::push_back(&mut v, 100);
+            v
+        });
+        reserve_config::set_interest_rate_aprs(&mut builder, {
+            let v = vector::empty();
+            vector::push_back(&mut v, 3153600000);
+            vector::push_back(&mut v, 3153600000 * 2);
+
+            v
+        });
+
+        sui::test_utils::destroy(config);
+        let config = reserve_config::build(builder, test_scenario::ctx(scenario));
+
+        reserve::create_for_testing<P, TEST_AUSD>(
+            config,
+            5,
+            6,
+            decimal::from(1),
+            0,
+            0,
+            0,
+            decimal::from(0),
+            decimal::from(2),
+            0,
+            test_scenario::ctx(scenario)
+        )
+    }
+
     #[test_only]
     fun reserves<P>(scenario: &mut Scenario): vector<Reserve<P>> {
         let v = vector::empty();
         vector::push_back(&mut v, sui_reserve(scenario));
         vector::push_back(&mut v,  usdc_reserve(scenario));
         vector::push_back(&mut v,  usdt_reserve(scenario));
         vector::push_back(&mut v,  eth_reserve(scenario));
+        vector::push_back(&mut v,  ausd_reserve(scenario));
```

Comment:
nit: can you make a new test_is_looped_2 function and not touch the existing one? just so im more confident in the code

Reply:
done